Libplanet.Tests/TestUtils.cs

```diff
@@ -395,5 +396,20 @@ public static string ToString(BitArray bitArray)
 
             return Hashcash.Hash(new Codec().Encode(dict));
         }
+
+        public static PrivateKey GeneratePrivateKeyOfBucketIndex(Address tableAddress, int target)
+        {
+            var table = new RoutingTable(tableAddress);
+            var count = 0;
+            PrivateKey privateKey;
+            do
+            {
+                count++;
```

Comment:
What is this counter used for?

Reply:
Was used to count total tries for logging purpose, and it's not used anymore. I'll remove it thanks.